workers/unity/Assets/Playground/Scripts/Player/ProcessScoresSystem.cs

```diff
@@ -0,0 +1,38 @@
+using Generated.Playground;
+using Improbable.Gdk.Core;
+using Unity.Collections;
+using Unity.Entities;
+
+namespace Playground
+{
+    [UpdateInGroup(typeof(SpatialOSUpdateGroup))]
+    class ProcessScoresSystem : ComponentSystem
+    {
+        private struct ScoringData
+        {
+            public readonly int Length;
+            public ComponentDataArray<SpatialOSLauncher> Launcher;
+
+            [ReadOnly]
+            public ComponentArray<CommandRequests<Generated.Playground.Launcher.ScoreIncrease.Request>> CommandRequests;
+        }
+
+        [Inject] private ScoringData scoringData;
+
+        protected override void OnUpdate()
+        {
+            // Process all requests for updating player scores.
+            // Increase the respective players' scores.
```

Comment:
I don't think the comment is needed, the system is nice and short. :)

Reply:
My personal preference is to keep it -- but I agree it's best to be consistent and remove it for simple systems.